src/coreclr/gc/gcpriv.h

```diff
@@ -5555,6 +5584,9 @@ class heap_segment
     }
 
     void thread_free_obj (uint8_t* obj, size_t s);
+
+    PTR_heap_segment    prev_free_region;
+    region_free_list*   containing_free_list;
 #else //USE_REGIONS
```

Comment:
move these up to be next to the other data members? ie, after free_list_tail

Reply:
Ok - done.